FORGE/JOANAESTEVES/climate_vpd.py

```diff
@@ -0,0 +1,80 @@
+# We will need myFunctions file
+import myFunctions as mf
+
+# To parse out all the information, we use the csv module
+import csv
+
+# Read the csv file and store climate information
+with open("Climate_hisafe_knmievaluation_daily_from_01-01-1981_to_31-12-2010.csv", "r") as csv_file:
+    csvreader = csv.reader(csv_file)
+    clim_info = list(csvreader)
+
+# Create a new list and avoid reading the header
+lst_vpd = ['',"vpd"]
+
+# Populate the list with the vpd function's results
+lst_lines = len(clim_info)
+for i in range(lst_lines - 2):
+    Tmax = float(clim_info[i+2][3])
+    Tmin = float(clim_info[i+2][4])
+    RHmean = float(clim_info[i+2][5])
+    vpd = mf.calculate_vpd(Tmax, Tmin, RHmean)
+    vpd = round(vpd,2)
+    lst_vpd.append(vpd)
+
+# Create a new csv file in our folder - it will have an extra column with the vpd values
+with open("Climate_hisafe_knmievaluation_daily_from_01-01-1981_to_31-12-2010_vpd.csv", "w", newline='') as new_csvfile:
```

Comment:
add prefix to filename to save it in your folder, i.e "forge/joanaesteves/climate_hisafe ..."